beamerthemeuulm.sty

```diff
@@ -149,11 +160,19 @@
 % table of contents
 % ---------------------------------------------
 \AtBeginSection{
-    \begin{frame}{\thesection.~\insertsection}
+    \ifsectiontitleslide
+        \ifsectionoverview
+        \begin{frame}{\thesection.~\insertsection}
         \setlength{\parskip}{0ex}
         \vfill
         \tableofcontents[currentsection,hideothersubsections]
-    \end{frame}
+        \end{frame}
+        \else
+        \begin{frame}
+            \Huge\textbf{\thesection.~\insertsection}
```

Comment:
Why using `\Huge` and not a predefined beamer font?
I don't have anything against that, but is there any special reason besides "it should be really large?"

Reply:
Isn't `\Huge` a predefined beamer font?

Reply:
Yes sorry, i formulated the question wrong. With "predefined beamer font" i meant we could use `\setbeamerfont` and `\usebeamerfont` to not hardcode `\Huge` but use something with a semantic meaning:

```latex
\setbeamerfont{section title}{size=\Huge, series=\bfseries}`
```

using it with `\usebeamerfont{section title}` (or with a star to clear all other attributes).

If you don't like `section title` (which is a font already defined by beamer) feel free to choose any other name (`\setbeamerfont` can create new fonts as well). Maybe a look at the beamer appearance cheat-sheet is worth the time: <https://www.cpt.univ-mrs.fr/~masson/latex/Beamer-appearance-cheat-sheet.pdf>.

Reply:
That sounds good, I will use the beamerfont `section title`.